<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Vehicle Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="filterEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>200</width>
      <height>25</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Search Make...</string>
    </property>
   </widget>
   <widget class="QComboBox" name="makeCombo">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="vehicleTypeCombo">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Car</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Jeep</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>SUV</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Truck</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="modelCombo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>10</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="yearCombo">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="themeCheckBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>70</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Dark</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addVehicleButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>45</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteVehicleButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>45</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>45</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>45</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Load</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exportCSVButton">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>45</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Export CSV</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>45</y>
      <width>80</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QTableView" name="vehicleTableView">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>700</width>
      <height>360</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="importCSVButton">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>45</y>
      <width>100</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Import CSV</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
